src/viperleed/calc/classes/rparams/rparams.py

```diff
@@ -692,10 +691,24 @@ def getFortranMpiComp(self, comp='auto'):
             self.FORTRAN_COMP_MPI = ['mpiifort -Ofast', '']
             _LOGGER.debug('Using fortran compiler: mpiifort')
         elif found == 'mpifort':
-            self.FORTRAN_COMP_MPI = [
-                'mpifort -Ofast -no-pie -fallow-argument-mismatch',
-                ''
-                ]
+            # check for the mpifort version
+            mpifort_call = 'mpifort -Ofast'
+            try:  # Add version-dependent CLI args
+                mpifort_version = fortran_utils.get_mpifort_version()
+            except fortran_utils.FortranCompilerError:
+                _LOGGER.warning(
+                    'mpifort version could not be determined automatically. '
+                    'mpifort versions >= 10.0 may need the '
+                    '"-fallow-argument-mismatch" or "-std=legacy" flags to '
+                    'compile the TenseErLEED structure-search code. '
+                    'If an error occurs, please check the mpifort version and '
+                    'adapt the FORTRAN_COMP parameter as required.'
+                    )
+            else:
+                mpifort_call += ('' if mpifort_version < '10.0'
+                                 else ' -fallow-argument-mismatch'
+                                )
```

Comment:
```suggestion
                                 else ' -fallow-argument-mismatch')
```

Here, since the parenthesized part is not on a new line, the closing parenthesis goes on the last line.